src/targets/c/libcurl/fixtures/multipart-form-data.c

```diff
@@ -8,6 +8,6 @@ struct curl_slist *headers = NULL;
 headers = curl_slist_append(headers, "Content-Type: multipart/form-data; boundary=---011000010111000001101001");
 curl_easy_setopt(hnd, CURLOPT_HTTPHEADER, headers);
 
-curl_easy_setopt(hnd, CURLOPT_POSTFIELDS, "-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"foo\"\r\n\r\nbar\r\n-----011000010111000001101001--\r\n");
```

Comment:
These multipart payloads also no longer have trailing `\r\n`'s and that's also okay.